docs/fundamentals-and-concepts/stellar-data-structures/operations-and-transactions.mdx

```diff
@@ -5,32 +5,34 @@ sidebar_position: 40
 
 All operations fall into one of three threshold categories: low, medium, or high, and each threshold category has a weight between 0 and 255 (which can be determined using set_options). Thresholds determine what signature weight is required for the operation to be accepted. For example, let’s say an account sets the medium threshold weight to 5. If the account wants to successfully establish a trustline with the `changeTrust` operation, the weight of the signature(s) must be greater than or equal to 5.
 
-To learn more about signature weight, see the [Signature and Multisignature Encyclopedia Entry](../../encyclopedia/signatures-multisig)
+To learn more about signature weight, see the [Signature and Multisignature Encyclopedia Entry](../../encyclopedia/signatures-multisig).
 
-View a comprehensive list of Stellar operations and their threshold levels in the [List of Operations section](../list-of-operations)
+View a comprehensive list of Stellar operations and their threshold levels in the [List of Operations section](../list-of-operations).
 
 ### Transactions
 
 The Stellar network encodes transactions using a standardized protocol called External Data Representation (XDR). You can read more about this in our [XDR Encyclopedia Entry](../../encyclopedia/xdr).
 
 Accounts can only perform one transaction at a time.
 
-Transactions comprise a bundle of between 1-100 operations and are signed and submitted to the ledger by accounts. Transactions always need to be authorized by the source account’s public key to be valid, which involves signing the transaction object with the public key’s associated secret key. A transaction plus its signature(s) is called a transaction envelope.
+Transactions comprise a bundle of between 1-100 operations (except smart contract transactions, which can only have one operation per transaction) and are signed and submitted to the ledger by accounts. Transactions always need to be authorized by the source account’s public key to be valid, which involves signing the transaction object with the public key’s associated secret key. A transaction plus its signature(s) is called a transaction envelope.
 
 A transaction may need more than one signature- this happens if it has operations that affect more than one account or if it has a high threshold weight. Check out the [Signature and Multisignature Encyclopedia Entry](../../encyclopedia/signatures-multisig) for more information.
 
 Transactions are atomic. Meaning if one operation in a transaction fails, all operations fail, and the entire transaction is not applied to the ledger.
 
 Operations are executed for the source account of the transaction unless an operation override is defined.
 
+Smart contract transactions also go through a simulation process where developers can test how the transaction would be executed on the network using the RPC endpoint `simulateTransaction`. Read more in the [Soroban docs](https://soroban.stellar.org/api/methods/simulateTransaction).
```

Comment:
```suggestion
Smart contract transactions also go through a simulation process where developers can test how the transaction would be executed on the network using the RPC endpoint `simulateTransaction`. Read more in the [Soroban docs](https://soroban.stellar.org/docs/fundamentals-and-concepts/interacting-with-contracts#transaction-simulation).
```

Maybe this would be a better link for the transaction simulation stuff? Not sure